appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.60.1",
+    "version": "0.61.0",
```

Comment:
I definitely like splitting this up into small chunks, but I prefer to consolidate breaking changes into one package update to some extent. So I would vote we merge this PR, but hold up on releasing until you get a few more changes in